prime-router/src/main/kotlin/serializers/CsvSerializer.kt

```diff
@@ -212,7 +212,7 @@ class CsvSerializer(val metadata: Metadata) {
                             element.csvFields.map { field ->
                                 val value = report.getString(row, element.name)
                                     ?: error("Internal Error: table is missing ${element.fieldMapping} column")
-                                element.toFormatted(value, field.format)
+                                element.toFormatted(value, field.format, report.schema)
```

Comment:
Consider moving the error handling logic here as a try catch, exposing it earlier in the flow. This would also let you keep the `toFormated` function signature the same.